plugins/rtmp-services/data/services.json

```diff
@@ -2515,6 +2515,73 @@
                 "max video bitrate": 7200,
                 "max audio bitrate": 196
             }
+        },
+        {
+            "name": "Shareplay.tv",
+            "common":true,
```

Comment:
Please do not set common to true.
The decision to set this to true would be made by the OBS Project.

Reply:
okey